resources/js/views/settings/users/View.vue

```diff
@@ -447,13 +489,18 @@ export default {
           this.$store.dispatch('session/logout').then(() => {
             this.$router.push({ name: 'home' });
           });
+          Base.error(error, this.$root, error.message);
         } else if (error.response && error.response.status === env.HTTP_UNPROCESSABLE_ENTITY) {
           this.errors = error.response.data.errors;
```

Comment:
if there is an error with the roles field, you could trigger a reload of the roles. I did something similar on my open PR

Reply:
So should I change here something?